File: yunj/level1/34_정수 내림차순으로 배치하기.java
import java.util.Arrays;
import java.util.Comparator;
import java.util.List;

class Solution {
    public long solution(long n) {
        long answer = 0;
        String[] numStr = String.valueOf(n).split("");
        List<String> al = Arrays.asList(numStr);
        al.sort(Comparator.reverseOrder());
        numStr = al.toArray(new String[0]);
        answer = Long.parseLong(String.join("",numStr));
        return answer;
    }
}

File: sangjin/level1/34_정수_내림차순으로_배치하기.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

long long solution(long long n) {
    long long answer = 0;
    vector<int> arr;
    
    while(n != 0){
        arr.push_back(n%10);
        n /= 10;
    }
    
    sort(arr.begin(), arr.end());
    reverse(arr.begin(), arr.end());

    for(int i = 0; i < arr.size(); i++){
        answer *= 10;
        answer += arr[i];
    }
    
    return answer;
}